### PR TITLE
Upload drops: create view with drag & drop support

### PR DESCRIPTION
**Pull request by umschaudhary**
New design for upload drop from files:.

Looks like this now : 
Screen Shot 2022-07-15 at 01 50 03

### CODE REVIEW

File: frontend/src/features/Drops/CreateDrop.jsx [hunk 1]
```diff
@@ -5,15 +5,20 @@ import { useSelector, useDispatch } from 'react-redux';
...
     const navigate = useNavigate();
     const dispatch = useDispatch();
 
+    const onDrop = (files) => {
+        setFile(files[0])
+    }
+
     useEffect(() => {
         if (isFetching) {
             console.log("FETCHING!")
```
Comment (fc-anjos) [inline frontend/src/features/Drops/CreateDrop.jsx:24]: Remove this console.log. 
Do we need some sort of spinner here?

File: frontend/src/features/Drops/components/FileUpload/index.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React, { useCallback } from 'react'
+import { useDropzone } from 'react-dropzone'
+
+const FileUpload = ({ onDropFile }) => {
+
+    const onDrop = useCallback(files => {
+        onDropFile(files)
+    }, []);
```
Comment (fc-anjos) [inline frontend/src/features/Drops/components/FileUpload/index.jsx:8]: Do we need files on the dependency array here to re-evaluate onDrop based on the files being changed?
Please test and let me know

**Review by fc-anjos — CHANGES_REQUESTED** [review]

File: frontend/src/features/Drops/CreateDrop.jsx (hunk 1 above)
Comment (umschaudhary) [reply frontend/src/features/Drops/CreateDrop.jsx]: Removed. We don't need the spinner. It's already taken care on the Submit button.

File: frontend/src/features/Drops/components/FileUpload/index.jsx (hunk 2 above)
Comment (umschaudhary) [reply frontend/src/features/Drops/components/FileUpload/index.jsx]: Tested & implemented accordingly.

**Pull request merged**